compiler/one-cmds/one-quantize

```diff
@@ -226,6 +226,28 @@ def _get_parser():
         help='convert quantized model to another-typed quantized model (ex: int8 -> uin8).'
     )
 
+    # arguments for ampq option
+    ampq_quant_group = parser.add_argument_group('arguments for ampq option')
+    # ampq
+    ampq_quant_group.add_argument(
+        '--ampq', action='store_true', help='quantize model using ampq solver.')
+
+    # ampq_qerror_ratio
+    ampq_quant_group.add_argument(
+        '--ampq_qerror_ratio', type=str, help='quantization error ratio ([0, 1])')
+
+    # ampq_algorithm
+    ampq_quant_group.add_argument(
+        '--ampq_algorithm',
+        type=str,
+        help='one of bisection_auto, bisection_true, bisection_false')
```

Comment:
Q) why not use something like algorithm name? just `bisection`, ... and whatever name? using `auto`, `true` and `false` seems strange.

Reply:
We can split the arguments, but I'm not sure which one is better.

as-is
```
ampq_algorithm= # bisection_auto, bisection_true, bisection_false
```

split
```
ampq_algorithm=bisection
bisection_type= # auto, true, false
```


@stamalakhov auto seems ok, but `bisection_true` and `bisection_false` look not natural. How about naming `bisection_i16_front` or `bisection_i16_back` ?

Reply:
For me
```
ampq_algorithm=bisection
bisection_type= # auto, true, false
```
looks better.

`isection_i16_front or bisection_i16_back` also looks nice then just `true` or `false`. 
`bisection_false` gives me that `do not use bisection...` and this doesn't look logical.

Reply:
Assembling the two options, the below interface looks good to me :)

```
ampq_algorithm=bisection
bisection_type= # auto, i16_front, i16_back
```